activemerchant.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |s|
   s.add_dependency("offsite_payments", "~> 2.0.0")
 
   s.add_development_dependency('rake')
+  s.add_development_dependency('test-unit', '~> 1.2.3')
```

Comment:
We could, I first one to merge this, and try upgrading test-unit and mocha (also outdated) afterwards.